_includes/site-footer.html

```diff
@@ -0,0 +1,80 @@
+<footer class="usa-footer">
```

Comment:
10. Here we have the basic header with updated links and logos. This adds the seal and the agency name similar to how USWDS has it setup on the [Footer component page](https://designsystem.digital.gov/components/footer/)